src/Contao/Hooks/LeafletLayerCallback.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Contao Leaflet MetaModels integration.
+ *
+ * @package    contao-leaflet-metamodels
+ * @author     Cliff Parnitzky <contao@cliff-parnitzky.de>
+ * @copyright  2015-2020 netzmacht David Molineus
+ * @license    LGPL 3.0-or-later https://github.com/netzmacht/contao-leaflet-metamodels/blob/master/LICENSE
+ * @filesource MetaModels\CoreBundle\Contao\Hooks\ModuleCallback
+ */
+
+namespace Netzmacht\Contao\Leaflet\MetaModels\Contao\Hooks;
+
+use MetaModels\CoreBundle\Contao\Hooks\AbstractContentElementAndModuleCallback;
+
+/**
+ * This class provides callbacks for tl_module.
+ */
+class LeafletLayerCallback extends AbstractContentElementAndModuleCallback
```

Comment:
I use the naming convention of `[CamelizedTableName]DcaListener` for the class listener. The name is used already and also required the inheritance of a parent class. So I suggest do rename it to `MetaModelsLeafletLayerDcaListener` instead.

Reply:
Renamed `Netzmacht\Contao\Leaflet\MetaModels\Contao\Hooks\LeafletLayerCallback` to `Netzmacht\Contao\Leaflet\MetaModels\EventListener\Dca\MetaModelsLeafletLayerDcaListener`